### ISSUE TITLE
lldpd doesn't output JSON

### ISSUE THREAD

**Issue by deltasierra** [open-switch/opx-platform-config#31]
```lldpctl -f json doesn't return JSON format.

root@l3:/# lldpctl -f json
-------------------------------------------------------------------------------
LLDP neighbors:
-------------------------------------------------------------------------------
Interface:    e101-012-0, via: LLDP, RID: 5, Time: 39 days, 09:21:21
  Chassis:
    ChassisID:    mac 00:0e:1e:90:ff:10
  Port:
    PortID:       mac 00:0e:1e:90:ff:10```



lldpctl -f xml works correctly 

```root@compute001:~# lldpctl -f json
{
  "lldp": {
    "interface": [
      {
        "eno1": {
          "vlan": {
            "vlan-id": "2052",
            "pvid": true
          },
          "via": "LLDP",
          "rid": "1",
          "age": "46 days, 01:33:25",
          "ppvid": {
            "supported": true,
            "value": "2052",
            "enabled": true
          },


root@s1:~# dpkg-query -l 'lldp*'

lldpd                                  0.7.11-2+deb8u1          amd64                    implementation of IEEE 802.1ab (LLDP)

**Comment by jeff-yin** [comment]
Since OPX 2.1 is based on Debian Jessie, the version of lldpd that is included does not yet support JSON output. 
http://ftp-master.metadata.debian.org/changelogs//main/l/lldpd/lldpd_0.7.11-2+deb8u1_changelog

You can try the Jessie-Backports version, which does support JSON output:
https://packages.debian.org/jessie-backports/lldpd
http://ftp-master.metadata.debian.org/changelogs//main/l/lldpd/lldpd_0.9.5-1~bpo8+1_changelog

**Comment by jpsenior** [comment]
This is something I would hope see fixed in the OPX packages from the OPX maintainers, backported or not.  It is essential for a networking platform to include basic programmatic access for critical networking statistics.  Having to tool around this as an end-user is counter-productive to the adoption of the platform itself.

**Comment by jeff-yin** [comment]
Oh I wasn't ruling anything out, I was just pointing out why the JSON output wasn't showing up. We can certainly look into getting the backported version into the next release of OPX installers.

**Comment by jpsenior** [comment]
Thanks Jeff

**Comment by jeff-yin** [comment]
Issue has been fixed as of OPX 2.2.0.0-dev2
